src/Params.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import './Params.css';
+import Drawer from '@material-ui/core/Drawer';
+import Toolbar from '@material-ui/core/Toolbar';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+
+const Params = (props: any) => {
+    const classes = props.classes;
+    return (
+        <div className="params">
+            <Drawer
+                className={classes.drawer}
+                variant="permanent"
+                classes={{
+                    paper: classes.drawerPaper,
+                }}
+            >
+                <Toolbar />
+                <div className={classes.drawerContainer}>
+                    <List
```

Comment:
As the Params component is only a container for a particular param, it should not share implementation with them.  Such component should be defined as separate component. How and when to split components into separate parts, and when include them in the common file is not always black and white, but you can always as a question - if this component could live on its own in different part of application? List of workers can. ListItem on the other hand.. Maybe not. To not multiply tens of files to do simple list, it can be defined in one common file ListOfWorkers - for example. But it should be still declared as 2 separate components. Try to avoid putting too much logic into JSX. It should be as clean as possible, ideally only other components (or basic Tags) calls with params and eventually conditions if children should be rendered or not.